### DISCUSSION TITLE
Seeking Best Practices for Auto-Generating MDX Blog Posts for Streamlining Site Updates

### DISCUSSION THREAD

**Discussion by TheSethRose** [timlrx/tailwind-nextjs-starter-blog#759] [General]
Hey everyone! 👋

I'm trying to determine the most efficient way to auto-generate MDX blog files and automatically push them to GitHub, triggering a Vercel update. 🚀

I'd love to hear your recommendations for this process. 🛠️ If you have any insights or have come across alternative methods that might make this workflow smoother, please share your thoughts!

The goal is to set up a dependable workflow that keeps my site fresh with minimal manual effort. Using content pulled from somewhere or fed in via some process. 

API Connections? Vercel Integrations?

Can it automatically pull in Discussions through Giscus?

Thanks in advance for your help!

**Comment by TheSethRose** [comment]
@timlrx I saw the readme contained a link to your site with a description about automating creation. Any input / ideas / code to point me in the right direction?

**Reply by timlrx** [reply]
Hi @TheSethRose, I don't have any recommendations on automating creation - I just use github with vercel and push an updated markdown post every time I want to update the blog. 

If you are comfortable working with markdown in VSCode, I think that's a fine option. If you are collaborating with other people who might not be as comfortable, that is a little more tricky. You could get them to edit off a branch on github and modify the contents from GIthub's markdown editor. Or you could look at other CMS solutions to try and integrate it with the site - I don't have any recommendations for CMS solutions.
